apps/mobile/README.md

```
@@ -0,0 +1,176 @@
+		<string>en</string>
+		<string>es</string>
+	</array>
+
+    ...
+```
+
+### Adding Translations
+
+1. For each supported locale, add a new ARB file in `lib/l10n/arb`.
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)